### PR TITLE
Add sandbox telemetry for version 2

### PR DESCRIPTION
**Pull request by pvollan**
#### a31fca13ab11fdc8dbd646bf068cab390e939eb0
<pre>
Add sandbox telemetry for version 2
<a href="https://bugs.webkit.org/show_bug.cgi?id=247195">https://bugs.webkit.org/show_bug.cgi?id=247195</a>
rdar://101678178

Reviewed by Chris Dumez.

Add sandbox telemetry for version 2 for access that was implicitly allowed in version 1. These telemetry rules
will be removed once we have gathered this information.

* Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in:
* Source/WebKit/NetworkProcess/mac/com.apple.WebKit.NetworkProcess.sb.in:
* Source/WebKit/Shared/Sandbox/macOS/common.sb:

Canonical link: <a href="https://commits.webkit.org/256107@main">https://commits.webkit.org/256107@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/2cc34a91edbc18cfc9ce899c13ced7363b663d91

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/94705 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/3863 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/27582 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/104326 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/3931 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/32051 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/87019 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/100260 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/100372 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/2831 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/81067 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/29845 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/84760 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/84306 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/72734 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/38457 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/18120 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/36303 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/19399 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/40214 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/42048 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2011 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/42186 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/38632 "Passed tests") | |

### CODE REVIEW

File: WebKit.xcworkspace/xcshareddata/xcschemes/Everything up to WebKit.xcscheme [hunk 1]
```diff
@@ -152,6 +152,16 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
+      <BuildableProductRunnable
```
Comment (cdumez) [inline WebKit.xcworkspace/xcshareddata/xcschemes/Everything up to WebKit.xcscheme]: Is this change intentional? I don't understand what this part does.

**Review by cdumez — APPROVED** [review]

File: WebKit.xcworkspace/xcshareddata/xcschemes/Everything up to WebKit.xcscheme (hunk 1 above)
Comment (pvollan) [reply WebKit.xcworkspace/xcshareddata/xcschemes/Everything up to WebKit.xcscheme]: > Is this change intentional? I don't understand what this part does.

Ah, good catch, this was not intended, will remove.

Thanks for reviewing!
